doctr/models/detection/zoo.py

```diff
@@ -46,14 +46,20 @@ def _predictor(arch: Any, pretrained: bool, assume_straight_pages: bool = True,
                 f"{ROT_ARCHS}"
             )
 
-        _model = detection.__dict__[arch](pretrained=pretrained, assume_straight_pages=assume_straight_pages)
+        _model = detection.__dict__[arch](
+            pretrained=pretrained,
+            pretrained_backbone=kwargs.get("pretrained_backbone", True),
+            assume_straight_pages=assume_straight_pages,
+        )
     else:
         if not isinstance(arch, (detection.DBNet, detection.LinkNet)):
             raise ValueError(f"unknown architecture: {type(arch)}")
 
         _model = arch
         _model.assume_straight_pages = assume_straight_pages
 
+    kwargs.pop("pretrained_backbone", None)
```

Comment:
no need for the second arg since we don't capture the output

Reply:
mh .. i have tried this but without success.
However, I don't remember what exactly the error was :sweat_smile:

Reply:
oh perhaps it fails if the key doesn't exist without the default value